data/all ships.txt

```diff
@@ -147,3 +147,17 @@ shipyard "All Ships"
 	"Wasp"
 	"Water Bug"
 	"Winter Gale"
+
+
+
```

Comment:
You should probably add a comment here to describe this section. The roles of other sections are immediately obvious, but this section is just kinda "there". Something like

```
# Ships without categories are not displayed in Shipyards.
# Supplying a valid category allows these ships to be purchased.
```

Reply:
Comment added.